packages/pds/src/db/index.ts

```diff
@@ -55,20 +58,61 @@ export class Database {
       dialect: new PostgresDialect({ pool }),
     })
 
-    return new Database(db, 'pg', schema)
+    return new Database(db, { dialect: 'pg', pool, schema })
   }
 
   static memory(): Database {
     return Database.sqlite(':memory:')
   }
 
+  notify(channel: string, msg?: string) {
+    if (this.facets.dialect === 'pg') {
+      this.facets.pool.query(`NOTIFY ${channel}, '${msg}'`)
+    } else {
+      const cbs = this.listeners[channel]
+      if (cbs && cbs.length > 0) {
+        for (const cb of cbs) {
+          cb(msg)
```

Comment:
It's a super minor thing, but to avoid timing inconsistencies it might be worth putting this behind a `process.nextTick(cb, msg)`.  Here's an example:
```ts
let item = 'a'
await db.listenFor('x', (msg) => {
  item = msg
})
db.notify('x', 'b')
// It's nice if it's well-defined whether item is 'a' or 'b' here.
```